src/utils/array.js

```diff
@@ -0,0 +1,3 @@
+export const getIntersection = (array1, array2) => {
+  return array1.filter((value) => array2.includes(value));
```

Comment:
유틸 함수인만큼 시간 복잡도 개선을 위해 includes 메서드 대신 Set 이용해보면 어떨까요?

Reply:
```suggestion
  const uniqueSet = new Set(array2);
  return array1.filter((value) => uniqueSet.has(value));
```
오.. 여기서 set은 생각도 못해봤던 것 같네요..😮